app/src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -284,18 +278,13 @@ class Simpletask : ThemedNoActionBarActivity() {
         if (Constants.INTENT_START_FILTER == intent.action) {
             mainFilter.initFromIntent(intent)
             log.info(TAG, "handleIntent")
-            val extras = intent.extras
-            if (extras != null) {
-                for (key in extras.keySet()) {
-                    val value = extras.get(key)
-                    if (value != null) {
-                        log.debug(TAG, "%s %s (%s)".format(key, value.toString(), value.javaClass.name))
-                    } else {
-                        log.debug(TAG, "%s %s)".format(key, "<null>"))
-                    }
-
+            intent.extras?.let { extras ->
+                extras.keySet().map { Pair(it, extras[it]) }.forEach { (key, value) ->
+                    val debugString = value?.let { v ->
+                        "$v (${v.javaClass.name})"
+                    } ?: "<null>"
+                    log.debug(TAG, "$key $debugString")
                 }
-
```

Comment:
This is part "Brittle", part style. Let me know if you want to revert the style or the full thing.